av/frame.pyx

```diff
@@ -68,7 +68,9 @@ cdef class Frame(object):
 
     property dts:
         """
-        The decoding timestamp in :attr:`time_base` units for this frame.
+        The decoding timestamp copied from the :class:`~av.packet.Packet` that triggered returning this frame in :attr:`time_base` units.
```

Comment:
Is there a link to the upstream documentation we could include?

Reply:
I've included the link in the PR description: https://ffmpeg.org/doxygen/5.1/structAVFrame.html#aa52951f35ec9e303d3dfeb4b3e44248a